src/gen_cranberry/gc_board.cpp

```diff
@@ -67,7 +67,7 @@ static void gc_board_setup(struct gc_board* b){
     // Open Devices
     digitalWrite(_PIN_SEN_EN, HIGH);
     gc_dev_xbee_open();
-    gc_dev_solar_open();
+    gc_dev_apogeesp212_open();
```

Comment:
Based on the naming scheme for the other cranberry sensors, naming it gc_dev_sp212 looks like it would be more appropriate.